Runtime/Aptabase.cs

```diff
@@ -103,7 +103,7 @@ public static void OnApplicationFocus(bool hasFocus)
             }
             else
             {
-                Flush();
+                _dispatcher.FlushOrSaveToDisk();
                 StopPolling();
```

Comment:
I think that we need to make this return a `Task` as well:
```C#
public static Task Flush()
{
     return _dispatcher.Flush();
}
```
Also I think we should avoid `async void` methods, eg:
```C#
private static async void StartPolling(int flushTimer)
```